### PR TITLE
Dump messages to and from Google Pub/Sub

### PR DESCRIPTION
**Pull request by NickGrantONS**
# Motivation and Context
We might need to back up the contents of a 'queue' like we did during Census, but because we have killed Rabbit, and we're now using Google Pub/Sub, our tools no longer work.

# What has changed
Ported the old scripts over to work with Google Pub/Sub.

# How to test?
Locally, try publishing some messages to a topic, and then run the code like this...

` PUBSUB_EMULATOR_HOST=localhost:8538 python -m toolbox.message_tools.dump_subscription_to_files event_case-update_rh --project shared-project --destination banana`

and

`PUBSUB_EMULATOR_HOST=localhost:8538 python -m toolbox.message_tools.dump_files_to_topic event_case-update --project shared-project --source banana`

Then try following the README instructions to do it on the toolbox in GCP.

# Links
Trello: https://trello.com/c/T2KYzzBs

### CODE REVIEW

**Comment by NickGrantONS** [comment]
[Build PubSub subscription dump/restore scripts (5)](https://trello.com/c/T2KYzzBs/2693-build-pubsub-subscription-dump-restore-scripts-5)

File: toolbox/message_tools/dump_files_to_topic.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import argparse
+import glob
+import os
+
+from google.cloud import pubsub_v1
+
+CHUNK_SIZE = 1
```
Comment (liamtoozer) [inline toolbox/message_tools/dump_files_to_topic.py:7]: Think this can be removed from this script?

**Review by liamtoozer — APPROVED** [review]
Works great for me locally 👍

File: toolbox/message_tools/dump_files_to_topic.py (hunk 1 above)
Comment (NickGrantONS) [reply toolbox/message_tools/dump_files_to_topic.py]: removed 👍

**Review by LukeLoze1 — APPROVED** [review]

**Pull request merged**